test/Parser/interfaces.js

```diff
@@ -11,10 +11,7 @@ export default () => describe("Interfaces", () => {
     describe('Annotations', () => {
         valid`@foo interface A { }`;
         valid`@foo @bar interface A { }`;
-        valid`
-            @foo
-            @bar
-            interface A {}`
+        valid`@foo\n@bar\ninterface A {}`;
```

Comment:
@LegionMammal978 for ambiguity borks like the one which caused bork with initalizers. Because code blocks match `\s*` and if something has like `Node? \s*` and node isn't there it'll become `\s*\s*` which is ambiguous.